src/xenia/kernel/xam/profile_manager.cc

```diff
@@ -273,8 +273,9 @@ bool ProfileManager::MountProfile(const uint64_t xuid, std::string mount_path) {
 }
 
 bool ProfileManager::DismountProfile(const uint64_t xuid) {
-  return kernel_state_->file_system()->UnregisterDevice(
-      fmt::format("{:016X}", xuid) + ':');
+  const std::string xuid_as_string = fmt::format("{:016X}", xuid);
+  return kernel_state_->file_system()->UnregisterDevice("DASHUSER:\\" +
```

Comment:
I wouldn't add it that way. 
It will be problematic if something will mount profile with different path. Obviously dashboard loads currently used profile that way, but Xenia internally doesn't and applications can also mount it differently, so unmount will not be possible.

If you can remove it and I will tackle it later. Currently profile is loaded directly via host device, but in reality it should be mounted by ContentManager, however it contains few limitations currently that must be resolved